notifications_utils/markdown.py

```diff
@@ -101,7 +101,7 @@ def paragraph(self, text):
             # Restore http:// or https:// and strip out the <strong> tag that gets injected by
             # the `link`/`autolink` methods
             text = self._render_qr_data(
-                re.sub(r"<strong data-original-protocol='(https?://)'>(.*?)</strong>", r"\1\2", qr_code_contents)
+                re.sub(r"<strong data-original-protocol='(https?://|)'>(.*?)</strong>", r"\1\2", qr_code_contents)
```

Comment:
What is the pipe (`|`) doing here?

Reply:
It's saying that we might sometimes not get a protocol (eg in the case of `[]()` links), and when that happens we still need to strip the strong tags.